### PR TITLE
Adds project tinyobjloader

### PR DESCRIPTION
**Pull request by catenacyber**
cc @syoyo
Would you be interested in continuous fuzzing for tinyobjloader ?
I see there are already some afl files...
This PR enables it with oss-fuzz with a simple fuzz target about `ParseFromString`

### CODE REVIEW

File: projects/tinyobjloader/patch.diff [hunk 1]
```diff
@@ -0,0 +1,48 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```
Comment (inferno-chromium) [inline projects/tinyobjloader/patch.diff:1]: Please uploaded patches for these in upstream repo, we cant land diffs here.

**Review by inferno-chromium — CHANGES_REQUESTED** [review]

File: projects/tinyobjloader/patch.diff (hunk 1 above)
Comment (catenacyber) [reply projects/tinyobjloader/patch.diff:1]: https://github.com/tinyobjloader/tinyobjloader/pull/302

Comment (catenacyber) [reply projects/tinyobjloader/patch.diff]: @inferno-chromium it was merged, and the patch.diff is removed

**Review by inferno-chromium — APPROVED** [review]

**Comment by catenacyber** [comment]
> cmake shouldn't be needed.

Is cmake part of base-builder ?

If so, I guess it can be removed from a few Dockerfiles...

**Comment by inferno-chromium** [comment]
> > cmake shouldn't be needed.
> 
> Is cmake part of base-builder ?
> 
> If so, I guess it can be removed from a few Dockerfiles...

yes cmake, make, bazel etc are all now in either base-clang [https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-clang/Dockerfile#L23] or base-builder [that inherits from base-clang, https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/Dockerfile].

**Pull request merged**